src/screens/Grommet.js

```diff
@@ -112,6 +112,16 @@ const GrommetPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="drop">
```

Comment:
This should be under the options prop.
`options.drop.checkContainingBlock`

Reply:
ooh gotcha